modules/osdk-bundle-validate-about.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-bundle-validate.adoc
+
+:_content-type: CONCEPT
+[id="osdk-about-bundle-validate_{context}"]
+= About the bundle validate command
+
+While the Operator SDK `scorecard` command can run tests on your Operator based on a configuration file and test images, the `bundle validate` subcommand can validate local bundle directories and remote bundle images for content and structure.
+
+Bundle images are pulled from a remote registry and built locally before they are validated. Local bundle directories must contain Operator metadata and manifests. The bundle metadata and manifests should have a structure similar to the following bundle layout:
+
+.Example bundle layout
+[source,terminal]
+----
+./bundle
+  ├── manifests
+  │   ├── cache.my.domain_memcacheds.yaml
+  │   └── memcached-operator.clusterserviceversion.yaml
+  └── metadata
+      └── annotations.yaml
+----
+
+.`bundle validate` command syntax
```

Comment:
I feel like this could be on line 10, beneath the `bundle validate` description. It's a little awkward beneath the bundle structure layout, IMO.